src/parameters.toml

```diff
@@ -709,6 +709,12 @@ alias = "E_0_LD2004"
 value = 1.08e10
 type = "float"
 
+[Variable_time_scale_auctoconversion_coeff_alpha]
```

Comment:
There's a typo here, should be "autoconversion" not "auctoconversion"

Reply:
Oh no! who was the person reviewing it and was it me?